release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
+
+The upgrade to Kubernetes v1.22 removed the following deprecated
+`v1beta1` APIs. Migrate manifests and API clients to use the `v1` API
```

Comment:
This paragraph shouldn't be hard wrapped. Can you unwrap it? (You can use ctrl+j if you use Atom)